Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,48 @@
+#include "Person.h"
+#include "iostream"
+
+Person::Person() {}
+
+Person::~Person() {
```

Comment:
Just general formatting for the team, we have curly braces on a new line.

Reply:
Dat, where is this written down? Is it part of the auto styler?
![image](https://user-images.githubusercontent.com/65361192/204107219-079e0fb2-4caf-4aac-85be-e002c249bdf9.png)